lua/lvim/plugin-loader.lua

```diff
@@ -42,8 +42,19 @@ function plugin_loader.init(opts)
     vim.api.nvim_create_autocmd("User", { pattern = "LazyDone", callback = require("lvim.lsp").setup })
   end
 
-  vim.opt.runtimepath:append(lazy_install_dir)
```

Comment:
have you tried just doing as the docs suggest in lazy?

```lua
  vim.opt.runtimepath:prepend(lazy_install_dir)
```

it seems to work fine for me, here's the full diff

```diff
diff --git a/init.lua b/init.lua
index 3fdf2a4e..5931817a 100644
--- a/init.lua
+++ b/init.lua
@@ -1,9 +1,11 @@
-local base_dir = vim.env.LUNARVIM_BASE_DIR
+vim.env.LUNARVIM_BASE_DIR = vim.env.LUNARVIM_BASE_DIR
   or (function()
     local init_path = debug.getinfo(1, "S").source
     return init_path:sub(2):match("(.*[/\\])"):sub(1, -2)
   end)()
 
+local base_dir = vim.env.LUNARVIM_BASE_DIR
+
 if not vim.tbl_contains(vim.opt.rtp:get(), base_dir) then
   vim.opt.rtp:append(base_dir)
 end
diff --git a/lua/lvim/plugin-loader.lua b/lua/lvim/plugin-loader.lua
index ddb6c179..f24f279f 100644
--- a/lua/lvim/plugin-loader.lua
+++ b/lua/lvim/plugin-loader.lua
@@ -42,8 +42,8 @@ function plugin_loader.init(opts)
     vim.api.nvim_create_autocmd("User", { pattern = "LazyDone", callback = require("lvim.lsp").setup })
   end
 
-  vim.opt.runtimepath:append(lazy_install_dir)
-  vim.opt.runtimepath:append(join_paths(plugins_dir, "*"))
+  vim.opt.runtimepath:prepend(lazy_install_dir)
 
   pcall(function()
     -- set a custom path for lazy's cache
```